### DISCUSSION TITLE
Noto Sans - full set for multiple langs

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17549] [How to]
Gents I need to use one font for all possible languages in PDF reports. Thing is I am not sure how to use @font-face .
Can you guide me?

**Comment by mariuszkrzaczkowski** [comment]
This question has already been asked on github, search

**Reply by garryatbrighton** [reply]
Been searching. Non of the posted solutions working in 6.5. 
It is sort of hard to understand that you guys dont pay attention to ML functions of system. 
Whoever launching any sort of app for global audience, this is absolutely No 1 task to do.... or am I wrong?

**Comment by garryatbrighton** [comment]
added mPFD engine, and all is OK now. It was not possible to fix it with native YetiforcePDF. Default ML support for PDF/emails/WF should come with box.

**Comment by mariuszkrzaczkowski** [comment]
In PDF everything depends on the imported fonts, just like in the browser, but you have to be careful because the size of the PDF will be bigger.
I have prepared a package for NotoSans-fonts, after installing (Software configuration > Standard modules > Modules – Installation) it is available in the WYSIWYG editor

# [NotoSans-fonts.zip](https://github.com/user-attachments/files/16401551/NotoSans-fonts.zip)

# WYSIWYG 

![image](https://github.com/user-attachments/assets/b2a4f2e8-b47e-45aa-9a6f-acaa90375e0e)

# Sample HTML to PDF

```html
<p>DejaVu Sans: <span style="font-family:'DejaVu Sans'">请您说得慢些好吗？ 不但 而且 Can you speak slowly? Qǐng nín shuō de màn xiē hǎo ma? ąśżźćń</span></p>
<p>Noto Sans: <span style="font-family:'Noto Sans'">请您说得慢些好吗？ 不但 而且 Can you speak slowly? Qǐng nín shuō de màn xiē hǎo ma? ąśżźćń</span></p>
<p>Noto Sans SC: <span style="font-family:'Noto Sans SC'">请您说得慢些好吗？ 不但 而且 Can you speak slowly? Qǐng nín shuō de màn xiē hǎo ma? ąśżźćń</span></p>
```

# generated PDF

![image](https://github.com/user-attachments/assets/76b964e3-a4df-4478-96e6-6a4149e1ee76)
